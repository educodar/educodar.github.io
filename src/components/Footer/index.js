import React from "react"
import { GatsbyLink, PageFooter } from "muy"
import Typography from "@material-ui/core/Typography"
import { Container } from "@material-ui/core"

const Footer = ({ siteTitle }) => (
  <PageFooter bgcolor={"primary"} pb={3} pt={3}>
    <Container maxWidth={"md"}>
      <Typography paragraph={true}>
        2019-{new Date().getFullYear()} <strong>{siteTitle}</strong>
      </Typography>
      <GatsbyLink target={"_self"} to={"/codigo-de-conduta"}>
        CÃ³digo&nbsp;de&nbsp;Conduta
      </GatsbyLink>
    </Container>
  </PageFooter>
)

export default Footer
