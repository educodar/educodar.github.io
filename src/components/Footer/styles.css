footer {
  padding: 1em 2em;
}
