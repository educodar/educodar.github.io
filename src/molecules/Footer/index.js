import React from "react"
import "./footer.module.css"
import { Link } from "gatsby"

const Footer = () => (
  <footer>
    <Link to="/">Privacidade e Uso de Dados</Link>
    <Link to="/">Termos de uso</Link>
  </footer>
)

export default Footer
