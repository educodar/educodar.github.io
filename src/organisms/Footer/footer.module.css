.footer {
  composes: container from global;
  background-color: var(--primary-variant-color);
  display: flex;
  justify-content: space-between;
  width: 100vw;
}

.footer a:any-link,
.footer a:link,
.footer a:visited {
  color: white;
  font-weight: bold;
  text-decoration: none;
}
