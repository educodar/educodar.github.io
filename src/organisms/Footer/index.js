import React from "react"
import style from "./footer.module.css"

import ConductCode from "../../molecules/ConductCode"

const Footer = () => (
  <footer className={style.footer}>
    <div>
      2019 <strong>Educodar</strong>
    </div>
    {/*<Link to="/privacidade">Privacidade e Uso de Dados</Link>
    <Link to="/termos-de-uso">CÃ³digo de Conduta</Link>*/}
    <ConductCode />
  </footer>
)

export default Footer
