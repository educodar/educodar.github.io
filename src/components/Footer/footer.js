import React from "react"
import "./styles.css"

const Footer = () => (
  <footer>
    <p>ThoughtWorks</p>
  </footer>
)

export default Footer
