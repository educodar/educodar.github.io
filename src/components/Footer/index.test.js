import renderer from "react-test-renderer"
import React from "react"
import Footer from "./index"

describe("Page footer", () => {
  /**
   * @todo fix failing test on Footer(...): Nothing was returned from render
   */
  xit("should render correctly", () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
