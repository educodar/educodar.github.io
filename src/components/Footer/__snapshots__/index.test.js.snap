// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page footer should render correctly 1`] = `
<footer
  className="MuiBox-root MuiBox-root-2"
  itemScope={true}
  itemType="https://schema.org/WPFooter"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthMd"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
    >
      2019-
      2020
       
      <strong />
    </p>
    <a
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
      href="/codigo-de-conduta"
      onBlur={[Function]}
      onFocus={[Function]}
      target="_self"
    >
      Código de Conduta
    </a>
  </div>
</footer>
`;
